### ISSUE TITLE
Issue template not showing up while creating a new issue

### ISSUE THREAD

**Issue by Abhishek-kumar09** [checkstyle/checkstyle#9285]
### Expected

An issue teamplate is expected while creating a new issue, asking for whether to go for bug or new functionality issue template:
![image](https://user-images.githubusercontent.com/48255244/107945135-31312480-6fb5-11eb-9cba-4093e0bb7cfe.png)

### Actual

No such isssue template is produced while doing so. The user is redirected to a new blank issue creation page.

**Comment by pbludov** [comment]
Blocked by #9292

**Comment by strkkk** [comment]
fix is merged
